docs/Istio-on-karmada-different-network.md

```diff
@@ -0,0 +1,273 @@
+
+## Deploy Istio
+
+### Install istioctl
+
+Please refer to the [istioctl](https://istio.io/latest/docs/setup/getting-started/#download) Installation.
+
+### Prepare CA certificates
+
+Following the
+steps [plug-in-certificates-and-key-into-the-cluster](https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/#plug-in-certificates-and-key-into-the-cluster)
+to configure Istio CA.
+
+Replace the cluster name `cluster1` with `primary`, the output will looks like as following:
+
+```bash
+[root@vm1-su-001 istio-1.12.6]# tree certs/
+certs/
+├── primary
+│   ├── ca-cert.pem
+│   ├── ca-key.pem
+│   ├── cert-chain.pem
+│   └── root-cert.pem
+├── root-ca.conf
+├── root-cert.csr
+├── root-cert.pem
+├── root-cert.srl
+└── root-key.pem
+```
+
+### Install Istio on karmada-apiserver
+
+Export `KUBECONFIG` and switch to `karmada apiserver`:
+
+```bash
+export KUBECONFIG=$HOME/.kube/karmada.config
+kubectl config use-context karmada-apiserver
+```
+
+Create a secret `cacerts` in `istio-system` namespace:
+
+```bash
+kubectl create namespace istio-system
+kubectl create secret generic cacerts -n istio-system \
+    --from-file=certs/primary/ca-cert.pem \
+    --from-file=certs/primary/ca-key.pem \
+    --from-file=certs/primary/root-cert.pem \
+    --from-file=certs/primary/cert-chain.pem
+```
+
+Create a propagation policy for `cacerts` secret:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: cacerts-propagation
+  namespace: istio-system
+spec:
+  resourceSelectors:
+    - apiVersion: v1
+      kind: Secret
+      name: cacerts
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+EOF
+```
+
+Run the following command to install istio CRDs on karmada apiserver:
+
+```bash
+istioctl install
+```
+
+Karmada apiserver will not deploy a real istiod pod, you should press `ctrl+c` to exit installation
+when `Processing resources for Istiod`.
+
+```bash
+✔ Istio core installed                                                           
+- Processing resources for Istiod.  
+```
+
+### Install Istiod on member1
+
+1、Close Karmada's auto-sync feature when labeling a namespace,because I need to label the same namespace `istio-system`
+in the different clusters differently.
+
+Export `KUBECONFIG` and switch to `karmada host`:
+
+```bash
+export KUBECONFIG=$HOME/.kube/karmada.config
+kubectl config use-context karmada-host
+```
+
+Edit `karmada-controller-manager`deployment
+
+```bash
+kubectl edit deployment karmada-controller-manager -n karmada-system
+```
+
+add `--controllers=-namespace,*` in start command
+
+2、Set the member1's network
+
+Export `KUBECONFIG` and switch to `member1`:
+
+```bash
+export KUBECONFIG="$HOME/.kube/members.config"
+kubectl config use-context member1
+```
+
+Label namespace `istio-system`
+
+```bash
+kubectl label namespace istio-system topology.istio.io/network=network1

```

Comment:
I can't label `istio-system` with `topology.istio.io/network=network1`


![image](https://user-images.githubusercontent.com/17962021/172763792-09164e1b-6edd-4b49-b2cc-2142497253cb.png)
![image](https://user-images.githubusercontent.com/17962021/172763850-257781ea-ba56-4eac-b7d2-0c667c8251bb.png)
![image](https://user-images.githubusercontent.com/17962021/172764208-b0276feb-fcea-41a4-aaf6-6f4518e5d11a.png)

Reply:
sorry, I forgot that the namespace 'istio-system' on member1 and member2 was synced from karmada-apiserver.I have changed the way of labeling.
But,I have a question why karmada doesn't support to label namespace directly on member clusters if the namespace was synced from karmada-apiserver.